### PR TITLE
Enable workload identity

### PR DESCRIPTION
**Pull request by liamtoozer**
# Motivation and Context

We can start to move our RM apps to use Workload Identity in our GCP clusters. Each k8s service that interacts with GCP can now use its own service account. This PR adds service accounts to the acceptance tests manifest

# What has changed

* Added `acceptance-tests` service account to pod manifest
* Removed unused creds, along with a couple of duplicate envs

# How to test?

See Terraform PR

# Links

https://github.com/ONSdigital/census-rm-terraform/compare/enable-workload-identity?expand=1
https://github.com/ONSdigital/census-rm-performance-tests/compare/enable-workload-identity?expand=1
https://github.com/ONSdigital/census-rm-kubernetes/compare/enable-workload-identity?expand=1

[Trello](https://trello.com/c/Z5kmTtll/970-workload-identity-update-pub-sub-printfilesvc-acceptance-and-performance-tests-13)

### CODE REVIEW

**Comment by liamtoozer** [comment]
[Workload Identity - Update pub/sub, printfilesvc, Acceptance and Performance tests (13?)](https://trello.com/c/Z5kmTtll/970-workload-identity-update-pub-sub-printfilesvc-acceptance-and-performance-tests-13)

File: acceptance_tests_pod.yml [hunk 1]
```diff
@@ -80,11 +79,6 @@ spec:
         configMapKeyRef:
           key: sftp-qm-supplier-directory
           name: project-config
-    - name: SENT_PRINT_FILE_BUCKET
```
Comment (NickGrantX) [inline acceptance_tests_pod.yml:83]: I don't understand why we can delete this

**Review by NickGrantX — CHANGES_REQUESTED** [review]
`SENT_PRINT_FILE_BUCKET` is used in quite a few places

File: acceptance_tests_pod.yml (hunk 1 above)
Comment (liamtoozer) [reply acceptance_tests_pod.yml:83]: This was a duplicate config line, so I removed it to tidy up. It's still in here [on this line](https://github.com/ONSdigital/census-rm-acceptance-tests/pull/281/files#diff-3ceb2c4565bde5ee265b026900f4c716R111)

Comment (davidmort) [reply acceptance_tests_pod.yml:83]: i think this is a duplicate as the config exists at line 111 also

**Comment by davidmort** [comment]
> `SENT_PRINT_FILE_BUCKET` is used in quite a few places

@NickGrantONS this config is a duplicate by the looks of it

**Review by davidmort — APPROVED** [review]

**Review by NickGrantX — APPROVED** [review]

**Pull request merged**